tests/cpp/test_preseg_passes.cpp

```diff
@@ -982,4 +982,52 @@ TEST_F(PresegTest, TranslateRepeatToExpand5) {
   EXPECT_EQ(heuristic_param->scheduler_type, SchedulerType::PointWise);
 }
 
+// Repeating a broadcast ID. Repro of
+// https://github.com/NVIDIA/Fuser/issues/3682.
+TEST_F(PresegTest, TranslateRepeatToExpand6) {
+  auto fusion_ptr = std::make_unique<Fusion>();
+  Fusion& fusion = *fusion_ptr;
```

Comment:
```suggestion
  auto fusion = std::make_unique<Fusion>();
```
Nit: I tend to keep just one variable and replace `fusion.` with `fusion->` and `&fusion` with `fusion.get()`.

Reply:
I have a muscle memory of writing `fusion.`, so this is how I avoid thinking about which to use.